pkg/defaults/defaults.go

```diff
@@ -595,6 +595,9 @@ const (
 
 	// NetNsPath is the default path to the mounted network namespaces directory
 	NetNsPath = "/var/run/cilium/netns"
+
+	// EnableIternalTrafficPolicy is the default value for option.EnableInternalTrafficPolicy
+	EnableInternalTrafficPolicy = true
```

Comment:
Should there be a corresponding change in `daemon/`?

Reply:
There should! I should have marked this as draft, sorry.